src/presentation/components/Header.vue

```diff
@@ -1,18 +1,67 @@
 <template>
   <div class="header">
+    <div class="header__tabs">
+      <Tab v-for="tab in tabs" :title="tab.title" :path="tab.path" :icon="tab.icon" :isDefault="tab.isDefault" :isActive="tab.isActive" :onClose="tab.onClose" :key="tab.path"/>
```

Comment:
move each attribute to a separate line, and connect eslint plugin `plugin:vue/vue3-recommended` please

Reply:
@neSpecc Can we add prettier too? Or it's not neccessary?

Reply:
> @neSpecc Can we add prettier too? Or it's not neccessary?

Let's try

Reply:
> > @neSpecc Can we add prettier too? Or it's not neccessary?
> 
> Let's try

Better in a separate PR i guess